WordPress/Classes/Uploads/UploadsManager.swift

```diff
@@ -31,6 +34,9 @@ class UploadsManager: NSObject {
         self.uploaders = uploaders
 
         super.init()
+
+        /// This just makes sure the reachabilityObserver property is initialized here.
+        _ = reachabilityObserver
     }
 
     deinit {
```

Comment:
AFAIK You don't have to remove the observer since we are targeting ios > 9.0. From Apple docs:

> If your app targets iOS 9.0 and later or macOS 10.11 and later, you don't need to unregister an observer in its dealloc method. Otherwise, you should call this method or removeObserver(_:name:object:) before observer or any object specified in addObserver(forName:object:queue:using:) or addObserver(_:selector:name:object:) is deallocated.

https://developer.apple.com/documentation/foundation/notificationcenter/1413994-removeobserver

just in case that's the reason you are saving it as a property

Reply:
Apologies! I forgot to push my last commit, which made some changes to how UploadsManager uses reachability.